src/sandboxes/eth-sandbox/App.tsx

```diff
@@ -29,11 +30,6 @@ const StyledApp = styled.div`
 // Constants
 // =============================================================================
 
-declare global {
```

Comment:
This clashes with the wagmi `ethereum` object, which has this in it and appears to have global scope as well.